tools/explorer/tt_adapter/src/tt_adapter/ttir.py

```diff
@@ -50,7 +50,7 @@ def get_layout_attrs(tensor):
     ]
 
     if hasattr(tensor.type, "encoding") and tensor.type.encoding:
-        layout = tt.ir.LayoutAttr.getLayout(tensor.type)
+        layout = tt.ir.MetalLayoutAttr.getLayout(tensor.type)
```

Comment:
I think this might be outdated now that we deleted `ttir.py`, but the direct analog is https://github.com/tenstorrent/tt-mlir/blob/f4cfebae3b6b81f4802f4d225b65538130fea68b/tools/explorer/tt_adapter/src/tt_adapter/mlir.py#L351.